terraform/redirector.tf

```diff
@@ -0,0 +1,7 @@
+resource "aws_route53_record" "redirector" {
+  zone_id = aws_route53_zone.dandi.zone_id
+  name    = "" # apex
+  type    = "A"
+  ttl     = "300"
+  records = ["3.12.21.28"]
```

Comment:
could it be a name? (`dandiarchive.org`)

Reply:
The root domain name is specified by the `aws_route53_zone.dandi` resource. This is a subdomain record. The `name` for an apex subdomain is the empty string.

The `aws_route53_record` resource allows you to provide a FQDN as the `name`, but while that's more explicit, it's also more fragile if the root domain ever changes or the definitions are reused with a different root domain. Once a developer is familiar with Terraform and our infrastructure, I don't think it's confusing.